tests/framework/installer/ceph_installer.go

```diff
@@ -369,13 +369,30 @@ func (h *CephInstaller) UninstallRookFromMultipleNS(helmInstalled bool, systemNa
 			checkError(h.T(), err, "cannot remove serviceaccount rook-ceph-cluster")
 			assert.NoError(h.T(), err, "%s  err -> %v", namespace, err)
 
+			_, err = h.k8shelper.DeleteResource("-n", namespace, "serviceaccount", "rook-ceph-mgr")
```

Comment:
These lines aren't being executed in the test because of line 367 above with the version check. None of this block will be executed. I actually don't see how the existing service account and roles are being deleted in the tests. I would recommend removing this block from lines 367-395, and instead calling something like:
```
		roles := h.Manifests.GetClusterRoles(namespace, systemNamespace)
		_, err = h.k8shelper.KubectlWithStdin(roles, deleteFromStdinArgs...)

```

Reply:
ah, much simpler.  Retesting with that now.